packages/wonder-blocks-clickable/util/get-clickable-behavior.js

```diff
@@ -32,7 +32,9 @@ export default function getClickableBehavior(
     router?: any,
 ): React.ComponentType<React.ElementConfig<typeof ClickableBehavior>> {
     if (router && skipClientNav !== true && href && !isExternalUrl(href)) {
-        return ClickableBehaviorWithRouter;
+        // We cast to `any` here since the type of ClickableBehaviorWithRouter
+        // is slightly different from the return type of this function.
+        return (ClickableBehaviorWithRouter: any);
```

Comment:
This seems reasonable. Could you add a TODO or raise a ticket to change this to component that flips between the two properly (or at least to address this better)?